package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cornerstone-tools",
-  "version": "3.0.1",
+  "version": "4.0.0-rc4",
```

Comment:
on merge, this should be `4.0.0`.
Not that it matters much, as `semantic-release` goes off tags.

Reply:
Didn't we accidently use `4.0.0` on npm already? 😅

Reply:
Hah. 4.0.1 then :P